app/src/main/res/values-zh-rHK/strings.xml

```diff
@@ -10,7 +10,7 @@
     <string name="air_quality_level_4">不健康</string>
     <string name="air_quality_level_5">很不健康</string>
     <string name="air_quality_pm25">PM2.5</string>
-    <string name="air_quality_pm25_voice">細懸浮微粒</string>
+    <string name="air_quality_pm25_voice">微細懸浮粒子</string>
```

Comment:
Is it normal other strings than `hko_` ones are changed?

Reply:
I've changed the word for "particulate matter" and "mold" to HK usage in a few strings. But I could revert these and edit them on Weblate instead if it's easier for your review. There are no usable air quality and pollen sources for HK now anyway.